numpy/linalg/setup.py

```diff
@@ -38,7 +39,7 @@ def configuration(parent_package='', top_path=None):
         class numpy_linalg_lapack_lite(system_info):
             def calc_info(self):
                 info = {'language': 'c'}
-                if sys.maxsize > 2**32:
+                if sysconfig.get_config_var("SIZEOF_VOID_P") > 4:
```

Comment:
```suggestion
                if sysconfig.get_config_var("SIZEOF_SIZE_T") >= 8:
```
just wondering, if we should keep using `size_t`?  Although I am sure it doesn't matter in practice.